### PR TITLE
Import landing zone blueprints

### PR DESCRIPTION
**Pull request by morgante**
This imports the landing zone blueprints from CSR.

Git subtree is used to preserve history.

### CODE REVIEW

File: catalog/hierarchy/README.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# Org hierarchy blueprint
+
+[TOC]
```
Comment (bharathkkb) [inline catalog/hierarchy/README.md:3]: Is this expected to be rendered? I don't see it rendered in preview

File: catalog/hierarchy/README.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+## Prerequisites
+- Yakima cluster w/ Git Ops enabled
```
Comment (bharathkkb) [inline catalog/hierarchy/README.md:6]: Should we switch names throughout
```suggestion
- Config Controller cluster w/ Git Ops enabled
```

File: catalog/hierarchy/README.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+kpt cfg set hierarchy namespace <your namespace>
+kpt cfg set hierarchy org-id <your organization ID>
+```
+
+As part of step #9, you can run the following command to verify the state and existence of the resources
+```
+kubectl -n ${YOUR_NAMESPACE} get folders -o custom-columns=NAME:.metadata.name,ID:.status.name,STATUS_MESSAGE:.status.conditions[0].message,STATUS:.status.conditions[0].type
+```
+
+Owners:
```
Comment (bharathkkb) [inline catalog/hierarchy/README.md:33]: nit: emails in readmes. We should do directory based owners if we want to maintain these.

File: catalog/hierarchy/env-bu/shared-folders.yaml [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+---
+apiVersion: resourcemanager.cnrm.cloud.google.com/v1beta1
+kind: Folder
+metadata:
+  name: prod.shared
+  namespace: hierarchy # {"$kpt-set":"namespace"}
+spec:
+  displayName: shared
+  folderRef:
+    name: prod
```
Comment (bharathkkb) [inline catalog/hierarchy/env-bu/shared-folders.yaml:19]: nit: new line

File: catalog/hierarchy/simple/hierarchy.yaml [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+  namespace: hierarchy # {"$kpt-set":"namespace"}
+spec:
+  parentRef:
+    external: "123456789012" # {"$kpt-set":"org-id"}
+  config:
+    - shared
+    - dev
+    - prod
+    - qa
+
```
Comment (bharathkkb) [inline catalog/hierarchy/simple/hierarchy.yaml:19]: nit: extra line

File: catalog/landing-zone/Kptfile [hunk 6]
```diff
@@ -0,0 +1,192 @@
...
+          name: group-org-admins-member
+          pattern: group:${group-org-admins}
+          values:
+          - marker: ${group-org-admins}
+            ref: '#/definitions/io.k8s.cli.setters.group-org-admins'
+    io.k8s.cli.setters.group-billing-admins:
+      x-k8s-cli:
+        setter:
+          name: group-billing-admins
+          value: yakima@google.com
```
Comment (bharathkkb) [inline catalog/landing-zone/Kptfile:40]: should we swap this default email with a non existent one?

**Review by bharathkkb — APPROVED** [review]

File: catalog/hierarchy/README.md (hunk 1 above)
Comment (morgante) [reply catalog/hierarchy/README.md:3]: It looks like GitHub does the TOC automatically, so I'll remove this: https://github.blog/changelog/2021-04-13-table-of-contents-support-in-markdown-files/

File: catalog/landing-zone/Kptfile (hunk 6 above)
Comment (morgante) [reply catalog/landing-zone/Kptfile:40]: Good idea, done.

**Pull request merged**